.github/workflows/_docs.yml

```diff
@@ -194,7 +194,7 @@ jobs:
           retention-days: 14
           s3-bucket: doc-previews
           if-no-files-found: error
-          path: pytorch.github.io/docs/main/
```

Comment:
No, we probably want to keep this one as it's just for s3 bucket name prefix, and later do proper redirects in DrCI  (somehwere in https://github.com/pytorch/test-infra/tree/main/torchci )